acceptance/teardown/common/099_Archive_Logs.rb

```diff
@@ -1,15 +1,107 @@
-require 'pxp-agent/test_helper'
+require 'date'
 
-step 'Archive files created during tests' do
-  ## Copy file(s) from master
-  archive_file_from(master, '/var/log/puppetlabs')
+def file_glob(host, path)
+  result = on(host, "ls #{path}", :acceptable_exit_codes => [0, 2])
+  return [] if result.exit_code != 0
+  return result.stdout.strip.split("\n")
+end
+
+# This test is prefixed with zzz so it will hopefully run last.
```

Comment:
Is this a copy and pasted line? This test is not prefixed `zzz`.

Reply:
yeah it's copy pasta from `pe_acceptance_tests` I didn't want to reinvent the wheel

Reply:
I removed the comment